### DISCUSSION TITLE
Install on windows

### DISCUSSION THREAD

**Discussion by gwen210** [SaifAati/Geospatial-COSICorr3D#23] [Q&A]
I've tried numerous ways to install this on a windows machine but it keeps failing. The instructions on the readme file do not work. Has anyone been able to get this installed? I have both anaconda and miniconda installed along with git bash

**Comment by SaifAati** [comment]
@gwen210 I did not test the installation with conda environment on Windows OS. I highly recommend using Linux, but if that's not possible, I recommend trying the Docker installation on Windows.
